src/VerifyU2FController.js

```diff
@@ -46,8 +48,16 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
             var deferred = Q.defer();
             u2f.sign(appId, factorData.challenge.nonce, registeredKeys, function (data) {
               self.trigger('errors:clear');
+              var errorSummary = 'Error Code: ' + data.errorCode;
               if (data.errorCode && data.errorCode !== 0) {
-                deferred.reject({ responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}});
+                switch (data.errorCode) {
+                case 4:
```

Comment:
it might be a good idea to move this ```case``` statement to a common place, since I assume it will be used in Enroll and in Verify